rules/type-declaration/tests/Rector/ClassMethod/AddArrayReturnDocTypeRector/Fixture/return_php_doc_should_not_be_replaced_by_less_specific_type.php.inc

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Rector\TypeDeclaration\Tests\Rector\ClassMethod\AddArrayReturnDocTypeRector\Fixture;
+
+final class ReturnPhpDocShouldNotBeReplacedByLessSpecificType
```

Comment:
I think it is better to have `Skip` prefix for fixture that skipped:

class        : `SkipReturnPhpDoc1`
file name : `skip_return_php_doc1.php.inc`